.github/workflows/main.yml

```diff
@@ -1,28 +1,35 @@
 # Github actions CI file
 # Ryan Mulhall 2020
-# Builds and runs make check in two containers, one with mpi, one without
+# CI testing for the FRE-NCtools repo, builds and runs unit tests
+# container @ https://hub.docker.com/repository/docker/ryanmulhall/fre-nctools-base
 name: FRE-NCtools CI
 on: [push, pull_request]
 jobs:
-  build:
+  CI:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         with_mpi: ['','--with-mpi']
         enable_quad_precision: ['', '--enable-quad-precision']
-    env:
+    container:
+      image: ryanmulhall/fre-nctools-base
```

Comment:
Ryan, Would you know if the directory of an individual is required to be used here?

Reply:
Currently we don't have a central public container registry or dockerhub account and to run the action this way requires a registry to pull the image from. I believe this should change soon though as the container project moves along and then i can migrate it to whichever to solution we go with.